# FOUNDIN-PD Aim 1C: Multi-omic Sequencing of CRISPR-Edited iPSC-derived Dopaminergic Neuronal Cell Lines 
## Data Overview
21 samples from 10 PPMI participants of IPSC-derived dopaminergic neuron cell lines were sequenced with 10X Genomics Multiome Single-Cell ATAC + Gene Expression assay across two batches. 3 samples are replicates of PPMI3966 Clone 107, a control participant with no known PD mutation (i.e. SNCA, GBA, and LRRK2). The other 18 samples are obtained from 9 unique participants with a known PD mutation, where one sample is sequenced as is and the other is CRISPR-edited to revert the PD mutation to wildtype (i.e. isoSNCA, isoGBA, and isoLRRK2). A sample metadata file has been uploaded to the GitHub.

## File Structure
FOUNDIN-Suppl
|-- raw
|   |-- ASSAYS
|       |-- SCAT
|           |-- *_I1_001.fastq.gz
|           |-- *_R1_001.fastq.gz
|           |-- *_R2_001.fastq.gz
|           |-- *_R3_001.fastq.gz
|       |-- SCRN
|           |-- *_I1_001.fastq.gz
|           |-- *_I2_001.fastq.gz
|           |-- *_R1_001.fastq.gz
|           |-- *_R2_001.fastq.gz
|-- processed
|   |-- analysis
|       |-- PPMI*
|           |-- feature_linkage
|               |-- feature_linkage.bedpe
|               |-- feature_linkage_matrix.h5
|           |-- tf_analysis
|               |-- filtered_tf_bc_matrix
|                   |-- barcodes.tsv.gz
|                   |-- matrix.mtx.gz
|                   |-- motifs.tsv
|               |-- filtered_tf_bc_matrix.h5
|               |-- peak_motif_mapping.bed
|   |-- SCAT
|       |-- ATAC_aim1c.rds
|       |-- PPMI*
|           |-- cellranger_outs
|               |-- analysis
|                   |-- clustering
|                       |-- graphclust
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_2_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_3_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_4_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_5_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                   |-- dimensionality_reduction
|                       |-- lsa
|                           |-- lsa_components.csv  
|                           |-- lsa_features_selected.csv
|                           |-- lsa_variance.csv
|                           |-- lsa_dispersion.csv  
|                           |-- lsa_projection.csv
|                       |-- tsne
|                           |-- tsne_projection.csv
|                       |-- umap
|                           |-- umap_projection.csv
|               |-- atac_cut_sites.bigwig
|               |-- atac_fragments.tsv.gz
|               |-- atac_fragments.tsv.gz.tbi
|               |-- atac_peak_annotation.tsv
|               |-- atac_peaks.bed
|               |-- atac_possorted_bam.bam
|               |-- atac_possorted_bam.bam.bai
|               |-- cloupe.cloupe
|               |-- per_barcode_metrics.csv
|               |-- summary.csv
|               |-- web_summary.html
|   |-- SCRN
|       |-- GEX_aim1c.rds
|       |-- PPMI*
|           |-- cellranger_outs
|               |-- analysis
|                   |-- clustering
|                       |-- graphclust
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_2_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_3_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_4_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                       |-- kmeans_5_clusters
|                           |-- clusters.csv
|                           |-- differential_accessibility.csv
|                           |-- differential_expression.csv
|                   |-- dimensionality_reduction
|                       |-- pca
|                           |-- pca_components.csv  
|                           |-- pca_features_selected.csv
|                           |-- pca_variance.csv
|                           |-- pca_dispersion.csv  
|                           |-- pca_projection.csv
|                       |-- tsne
|                           |-- tsne_projection.csv
|                       |-- umap
|                           |-- umap_projection.csv
|               |-- filtered_feature_bc_matrix
|                   |-- barcodes.tsv.gz
|                   |-- features.tsv.gz
|                   |-- matrix.mtx.gz
|               |-- filtered_feature_bc_matrix.h5
|               |-- gex_molecule_info.h5  
|               |-- gex_possorted_bam.bam 
|               |-- gex_possorted_bam.bam.bai
|               |-- per_barcode_metrics.csv
|               |-- raw_feature_bc_matrix
|                   |-- barcodes.tsv.gz
|                   |-- features.tsv.gz
|                   |-- matrix.mtx.gz
|               |-- raw_feature_bc_matrix.h5
|               |-- cloupe.cloupe
|               |-- summary.csv
|               |-- web_summary.html